src/core/best-practices.md

```diff
@@ -0,0 +1,52 @@
+
+
+## Avoiding Slippage
+Slippage is the effect of the headset slipping on the participant’s head. This is almost inevitable if the participant moves its head or its facial muscles. Slippage will lead to reduced accuracy and can accumulate over time.
+
+### Split Your Experiment into Blocks
+If your experiment allows for small breaks in between, you should split it into blocks. For every block you should recalibrate (and validate) at the beginning to reset any accumulated slippage errors.
+
+### Choose the Right Pipeline
+The 2D detection and mapping pipeline has the best accuracy in perfect conditions (< 1° visual error). However, it is very sensitive to slippage. Chose the 2D pipeline if:
+- Your participants do not need to move their heads.
+- You can have very short blocks.
+- You want the best possible accuracy.
+
+The 3D pipeline will adapt itself over time to compensate for slippage errors. The tradeoff is that the best accuracy in perfect conditions is slightly below the accuracy of the 2D pipeline (1.5°–2.0° visual error). Note that slippage will still have an effect on very long recordings. Chose the 3D pipeline if:
+- Your participants need to move their heads.
+- You cannot split your experiment into small blocks.
+- You are fine with good accuracy.
+
+
+## Record Everything
+You do not have to make fixed decisions for an experiment, as Pupil Player allows for running pupil detection and gaze mapping again with different settings in an offline mode. In order for this to work best, you should record all the data that is needed to run different offline algorithms. Here are the steps to include in your recording:
+1. Start the recording.
+1. Ask the participant to slowly look around with the eyes for a couple of seconds to sample different gaze angles. This will allow the 3D model to stabilize at the start. Do this even if you are not using the 3D pipeline to allow for comparison in offline mode.
+1. Perform a calibration.
+1. Perform a validation. If the error is too high, restart the block.
+1. Perform your experiment.
+1. Perform another validation. This can give you an estimate of how much slippage error you accumulated during the experiment. If you are recording multiple blocks with a non-fixed calibration setup, you can also re-use the calibration session of the following block as post-validation for the previous block.
+1. Now you can stop the recording.
+
+
+## Participants
+The less movement you participants will exhibit during an experiment, the more accurate your results will be. This includes movements of head, but also of facial muscles. In order to minimiza unnecessary movement, you should:
+- Make sure the participant is in a comfortable position that they can maintain during the experiment.
+- Make sure the participant can read all instructions comfortable and does not need to squint.
+- Ask your participant to keep a relaxed facial expression during the experiment.
```

Comment:
```suggestion
```

I suggest to omit. This is really dependent on the experiment.

Reply:
What do you mean it depends on the experiment? For me moving facial muscles (around the eyes) while keeping a steady gaze will introduce noticeable error to the gaze mapping. This effect should be independent of the experiment. I saw that Tobii recommend something similar.

Reply:
It should be a note and note a recommendation. Some experiments depend on emotion of the participant, and in this case, researchers should know that this induces slippage. But they should not be limited by this.